doc/Comments.xml

```diff
@@ -408,21 +408,29 @@ not finished or temporary files.
 
 <Subsection Label="@BeginChunk">
 <Index Key="@BeginChunk"><C>@BeginChunk, @EndChunk, and @InsertChunk</C></Index>
-<Heading>@BeginChunk <A>name</A>, @EndChunk, and @InsertChunk <A>name</A></Heading>
-Text insider of a @BeginChunk / @EndChunk part will not be inserted into
+<Heading>@BeginChunk <A>name[ param1,...]</A>, @EndChunk, and @InsertChunk <A>name[ param1_value,...]</A></Heading>
+Text inside of a @BeginChunk / @EndChunk part will not be inserted into
 the final documentation directly. Instead, the text is stored in an internal buffer.
 
 That chunk of text can then later on be inserted in any other place by using the
 @InsertChunk <A>name</A> command.
 
+In the text, you can use <A>%param1</A> placeholders, which will then be overwritten
+by the <A>param1_value</A> parameters when the text is inserted. Different parameters
+may not contain spaces and must be separated by a comma. Different values may contain
```

Comment:
Also, "Different parameters" sounds weird to me. What are "Equal parameters"? Same for "Different values". The more I think about it, I'd completely rewrite this paragraph. Here is a suggestion:

> Chunks may optionally be parametrized, by specifying a comma-separated list of parameter names after the chunk name given to @BeginChunk. Each parameter name must be a valid GAP identifier (and hence may only contain letters, digits and underscores).
> When inserting a parametrized chunk, a comma-separated list of parameter values must follow the chunk name given to @InsertChunk. The number of parameter values must match the number of parameter names.
> When this is done, instead of directly inserting the chunk text, for every parameter <A>param</A>, every occurrence of the string <A>{param}</A> is substituted by the corresponding parameter value.